src/mpi/coll/algorithms/treealgo/treealgo.c

```diff
@@ -62,14 +65,162 @@ int MPIR_Treealgo_tree_create(int rank, int nranks, int tree_type, int k, int ro
     MPIR_FUNC_EXIT;
 
   fn_exit:
+    if (MPIR_CVAR_TREE_DUMP) {
+        char outfile_name[PATH_MAX];
+        sprintf(outfile_name, "%s%d.json", "tree", rank);
+        FILE *outfile = fopen(outfile_name, "w");
```

Comment:
Suggest `printf(stdout, "MPIR_Treealgo_tree_create: dumping %s, tree_type=%d\n", outfile_name, tree_type);`

Reply:
Added the printf